addons/google_calendar/models/calendar.py

```diff
@@ -267,9 +267,9 @@ def _google_values(self):
 
         attendees = self.attendee_ids
         attendee_values = [{
-            'email': attendee.partner_id.email_normalized,
+            'email': attendee.sudo().partner_id.email_normalized,
```

Comment:
I think the actual issue is we shouldn't be able to add private addresses as attendees, so the solution would be to exclude private addresses from the domain.

BTW, that approach wouldn't require tests.

Reply:
The domain is already limited, the issue is when the partner address type is changed.